install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -265,7 +265,6 @@ type ServerConfig struct {
 	DisableWorkspaceGarbageCollection bool              `json:"disableWorkspaceGarbageCollection"`
 	DisableCompleteSnapshotJob        bool              `json:"disableCompleteSnapshotJob"`
 	InactivityPeriodForReposInDays    *int              `json:"inactivityPeriodForReposInDays"`
-	ShowSetupModal                    *bool             `json:"showSetupModal"`
```

Comment:
Worth checking this is not configured in dedicated installations. If it is, we need to remove from there first.

Reply:
Going to revert this

Reply:
Would be great to remove this config as well. I believe it is only used in cells that we do not intend to update?

Reply:
I think the proper fix should for dedicated to ignore unknown config values (rather than fail) as it's gonna become impossible to keep track of this

Reply:
change reverted.